### PR TITLE
Added styling

### PR DESCRIPTION
**Pull request by VICTOR-CHUKWU**
Added styling to project

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,3 +1,260 @@
...
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  border-bottom: 1px solid rgb(230, 224, 224);
+  margin-bottom: 2rem;
+  width: 100%;
+}
```
Comment (omar-labana) [inline src/index.css]: - [ ] The `header` should match the rest of the layout and it should have a different background color, Kindly update your head to match the Zeplin design.
Your styling
image
Zeplin design
image

File: src/component/Header.js [hunk 2]
```diff
@@ -2,14 +2,14 @@ import React from 'react';
...
     <h2>Bookstore Victor</h2>
-    <div>
-      <Link to="/">
+    <div className="nav">
+      <Link to="/" className="link">
         {' '}
         <span>Books</span>
       </Link>
-      <Link to="/category">
+      <Link to="/category" className="link">
```
Comment (omar-labana) [inline src/component/Header.js]: - [ ] The `header` is missing the avatar icon. Also, the links are not properly styled/aligned, Kindly update your head to match the Zeplin design.
Your styling
image
Zeplin design
image

File: src/component/books/Book.js [hunk 3]
```diff
@@ -1,30 +1,44 @@
...
+                <h3>
+                  <span className="percentage"> 64%</span>
+                  <span className="completed">completed</span>
+                </h3>
+              </div>
+              <div className="chapter">
+                <h4>Current Chapter</h4>
+                <h3>chapter one: &quot; A Lesson Learned &quot;</h3>
+                <button type="button">Update progress</button>
+              </div>
```
Comment (omar-labana) [inline src/component/books/Book.js]: - [ ] Each `book` is missing the vertical separators `|` between `progress` and `Chapter info`. Also, the white space between them is a bit large, Kindly update your `book` component to match the Zeplin design.
Your styling
image
Zeplin design
image

- [ ] Each `book` is missing the `Author` name. Also, the vertical separators `|` between `Comment`, `Remove`, and `Edit` are missing, Kindly update your `book` component to match the Zeplin design(text colors are not matching).
Your styling
image
Zeplin design
image

File: src/component/books/AddBooks.js [hunk 4]
```diff
@@ -20,21 +20,23 @@ const AddBooks = () => {
...
+          value={title}
+        />
+        <input
+          type="text"
+          placeholder="category"
+          onChange={(e) => setCategory(e.target.value)}
+          value={category}
+        />
+        <button type="submit">submit</button>
+      </div>
```
Comment (omar-labana) [inline src/component/books/AddBooks.js]: - [ ] The form responsible for adding a new book isn't properly styled(input fields and the horizontal line before the form), Kindly update your book component to match the Zeplin design.
Your styling
image
Zeplin design
image

- [ ] Kindly, update the text of the form header to `ADD NEW BOOK`.
- [ ] Kindly, update the text of the submit button to `ADD BOOK`.
- [ ] The `Category` input should be a `select` field with are of the previously given categories.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @VICTOR-CHUKWU, 

Excellent work so far! 👌
You have implemented almost all the required features. ✔️ 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hi there 👋🏽  @VICTOR-CHUKWU  

Great work on making the changes requested by a previous reviewer 👏 
To highlight 👍🏽 
- You had added the avatar.
- You have no linter errors
- Each book has a vertical separator.
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] Please properly align the header links as the previous reviewer suggested. The books and category links should be closer to the logo
**_Your project_**
Screenshot 2021-11-11 at 18 44 27

**_Zeplin design_**
Screenshot 2021-11-11 at 18 47 20

- [ ] Also please make the form title and placeholder text the same as the one in the design. You should have a form title of ADD NEW BOOK, in uppercase letters and the first form input should have a placeholder text of Book title. It is currently Add new book in your design.
**_Your project_**
Screenshot 2021-11-11 at 18 52 57

**_Zeplin design_**

Screenshot 2021-11-11 at 18 52 48


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by VICTOR-CHUKWU** [comment]
kindly approve as this is my last chance

**Review by hiromataba — APPROVED** [review]
## Status: Approved :champagne: 
Hi @VICTOR-CHUKWU , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

### Optional Suggestion

- [ ] [OPTIONAL] Kindly use the project design `font-family`

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**